pkgs/development/python-modules/materialx/default.nix

```diff
@@ -53,10 +51,7 @@ buildPythonPackage rec {
   cmakeFlags = [
     (lib.cmakeBool "MATERIALX_BUILD_OIIO" true)
     (lib.cmakeBool "MATERIALX_BUILD_PYTHON" true)
-    # don't build MSL shader back-end on x86_x64-darwin, as it requires a newer SDK with metal support
-    (lib.cmakeBool "MATERIALX_BUILD_GEN_MSL" (
-      stdenv.hostPlatform.isLinux || (stdenv.hostPlatform.isAarch64 && stdenv.hostPlatform.isDarwin)
-    ))
+    (lib.cmakeBool "MATERIALX_BUILD_GEN_MSL" stdenv.hostPlatform.isLinux)
```

Comment:
This should be `|| stdenv.hostPlatform.isDarwin`, right?

Reply:
Should be fixed now.